tools/editor/editor_node.cpp

```diff
@@ -1,4 +1,4 @@
-/*************************************************************************/
+﻿/*************************************************************************/
```

Comment:
Ah, yeah... it's a BOM marker:
![damnit](https://cloud.githubusercontent.com/assets/7718100/14954573/ce66d296-1073-11e6-9d1b-28abedc5b9e1.png)

No idea why Qt Creator is adding it nor how to avoid it. I will remove it manually for now...